Directory.Build.props

```diff
@@ -12,9 +12,9 @@
     <RootDir>$(MSBuildThisFileDirectory)</RootDir>
 
     <!-- Defaults-->
-    <Major Condition="$(Major) == ''">1</Major>
-    <Minor Condition="$(Minor) == ''">0</Minor>
-    <Build_BuildNumber Condition="'$(Build_BuildNumber)' == ''">0.0</Build_BuildNumber>
+    <Major Condition="$(Major) == ''">99</Major>
+    <Minor Condition="$(Minor) == ''">99</Minor>
+    <Build_BuildNumber Condition="'$(Build_BuildNumber)' == ''">99.99</Build_BuildNumber>
```

Comment:
I'd prefer to keep it 1.0.0.0 by default unless overridden.. otherwise local build produces 99.99.99.99 version of nuget packages, which is unusual... I think industry-wide default is usually 1.0.0.0

Reply:
I don't think there's an industry standard for dev/local versions. And 1.0.0.0 is too easily confused with actual released versions - especially given that prior to my changes our actual official released versions all had an assembly version of 1.0.0.0. So I'd prefer to keep it as 99.99.99.99 since that's extremely clear that it's not a "real" released version.